index.html

```diff
@@ -17,14 +17,22 @@
             <a href="#"><h2>Bilal</h2></a>
         </div>
         <nav class="Menuicon">
-            <span class="material-symbols-outlined">
+            <span id="navicon" class="material-symbols-outlined">
                 menu
                 </span>
         </nav>
-        <ul class="menu">
-            <li><a class="menu_item" href="#portfolio">portfolio</a></li>
-            <li><a class="menu_item" href="#about">About</a></li>
-            <li><a class="menu_item" href="#contact">Contact</a></li>
+        <ul id="menu" class="menu">
+            <a href="#" id="cross" class="Menuiconcross"
+					><span
+						id="crossIcon"
+						style="color: white"
+						class="material-symbols-outlined"
+					>
+						close
+					</span></a>
+            <li class="lil"><a class="menu_item" href="#work">portfolio</a></li>
+            <li class="lil"><a class="menu_item" href="#aboutme">About</a></li>
+            <li class="lil"><a class="menu_item" href="#contact">Contact</a></li>
```

Comment:
- [ ] Your mobile menu does not disappear when a menu item is clicked. Please fixed that so that the user expereince is improved. because of this, the correct part of the page is not displayed as the menu still remains visible and on top 😓